lesson_71/HW_L71/models/user.js

```diff
@@ -0,0 +1,32 @@
+import mongoose from "mongoose";
+
+const Schema = mongoose.Schema;
+
+const userSchema = new Schema({
+  email: {
+    type: String,
+    required: true,
+    max: 254,
+    validate: {
+      validator: async function(email) {
```

Comment:
use can use unique modifier for this purpose: [https://mongoosejs.com/docs/schematypes.html#indexes](https://mongoosejs.com/docs/schematypes.html#indexes)